freqtrade/optimize/backtesting.py

```diff
@@ -115,11 +117,13 @@ def _generate_text_table(self, data: Dict[str, Dict], results: DataFrame) -> str
 
     def _store_backtest_result(self, recordfilename: Optional[str], results: DataFrame) -> None:
 
-        records = [(trade_entry.pair, trade_entry.profit_percent,
-                    trade_entry.open_time.timestamp(),
-                    trade_entry.close_time.timestamp(),
-                    trade_entry.open_index - 1, trade_entry.trade_duration)
-                   for index, trade_entry in results.iterrows()]
+        # columns = ["pair", "profit", "opents", "closets", "index", "duration",
```

Comment:
Seems like a debugging leftover, if not please write a proper comment

Reply:
Love a good Review!

moved this to the documentation (including a code-snipped for "manual" analysis"...)